icons.json

```diff
@@ -28,6 +28,13 @@
   }, {
     "name": "fullscreen-exit",
     "svg": "navigation/svg/production/ic_fullscreen_exit_48px.svg"
+  }, {
```

Comment:
can these icons be moved to the end of the list? Otherwise, it'll shift codepoints for existing icons, which we want to avoid.

Reply:
perhaps this should be briefly documented somewhere as well.

Reply:
Filed https://github.com/videojs/font/issues/42

Reply:
Done in https://github.com/videojs/font/pull/41/commits/5fa24d29ccb4da455ad6bb17e595a54ae57ea0f7